docs/linux/eduroam.md

```diff
@@ -17,4 +15,4 @@ Password | `<wachtwoord>`
 
 [^1]: Select from file... > Other locations (links) > Computer > navigeer naar /etc/ssl/certs en selecteer USERTrust_RSA_Certification_Authority.pem
 
-![WiFi instellingen Ubuntu 20.04](../assets/2004-wifi.png)
+![WiFi instellingen Ubuntu 20.04](/assets/2004-wifi.png)
```

Comment:
relatieve paden zijn handiger omdat de wiki dan in de toekomst verplaatst zou kunnen worden naar een subdirectory. Mkdocs geeft hier nu een warning over dus dat zal ik fixen

Reply:
En ik kom er direct achter dat het er een goede reden voor is, anders werkt het bij de include in post-installatie niet.. Dat is onhandig